web/templates/message/index.html.eex

```diff
@@ -0,0 +1,6 @@
+<div class="tc">
+  <label for="message-input" class="f5 b db mb2 mt4">Enter a message</label>
+  <input id="message-input" class="pa3 input-reset bn hl-bg-grey w-75"/>
+</div>
```

Comment:
I prefer it that way as well, but went for the other way around since it was like that in the wireframes. I thought we could wait for user feedback on this portion before switching it around if necessary.